Makefile

```diff
@@ -0,0 +1,91 @@
+all: jenkins prow mungegithub projects
+.PHONY: all
+
+jenkins:
+.PHONY: jenkins
+
+prow: prow-crd prow-config prow-images hook plank jenkins-operator deck horologium splice sinker commenter
+.PHONY: prow
+
+prow-crd:
+	oc apply -f cluster/ci/config/prow/prow_crd.yaml
```

Comment:
This shouldn't be updated often enough to justify a make target I think.

Reply:
nest it under something else? we want one `make` target for initial bringup at the top level

Reply:
Ok, then this is fine.